#! /bin/sh /usr/share/dpatch/dpatch-run
# Description: Hide useless help buttons in dialogs
# Author: Felix Geyer <debfx-pkg@fobos.de>
# Bug: http://www.virtualbox.org/ticket/6442

@DPATCH@
diff -Nur virtualbox-ose-3.1.6-dfsg.orig/src/VBox/Frontends/VirtualBox/src/extensions/QIDialogButtonBox.cpp virtualbox-ose-3.1.6-dfsg/src/VBox/Frontends/VirtualBox/src/extensions/QIDialogButtonBox.cpp
--- virtualbox-ose-3.1.6-dfsg.orig/src/VBox/Frontends/VirtualBox/src/extensions/QIDialogButtonBox.cpp	2010-03-26 11:09:50.000000000 +0100
+++ virtualbox-ose-3.1.6-dfsg/src/VBox/Frontends/VirtualBox/src/extensions/QIDialogButtonBox.cpp	2010-03-27 11:15:33.645861192 +0100
@@ -64,6 +64,9 @@
 
 void QIDialogButtonBox::setStandardButtons (StandardButtons aButtons)
 {
+#ifdef VBOX_OSE
+    aButtons &= ~QDialogButtonBox::Help;
+#endif
     QDialogButtonBox::setStandardButtons (aButtons);
     retranslateUi();
 }
